number_of_motors: 2
gear_ratio: [4, 1]
wheel_base: 0.8636
wheel_radius: 0.127
tire_deflection: 1.042
diameter_mod: 1.0